syweb/webclient/test/unit/event-handler-service.spec.js

```diff
@@ -120,7 +120,14 @@ describe('EventHandlerService', function() {
     };
     
     var notificationService = {
-        containsBingWord: function(user_id, display_name, bing_words) {
+        processEvent: function(ev) {
+            if (matrixService.config().muteNotifications) return;
+            if (testContainsBingWords) this.showNotification(1,1,1,1);
+            if (ev.type == 'm.room.member' && ev.content.membership == 'invite' && ev.state_key == testUserId) {
+                this.showNotification(1,1,1,1);
```

Comment:
I don't understand the inclusion of this logic in the mock ``notificationService``. The ``event-handler-service.spec.js`` file shouldn't be asserting that the ``notificationService`` is doing things. If the tests just assert ``processEvent`` is called, you should just mock this away and leave a stub. If the tests assert that ``notificationService`` is calling/doing other things in ``processEvent``, then the tests should be fixed to not be testing that here.